bliss/metrics.py

```diff
@@ -188,8 +145,13 @@ def match_by_locs(true_locs, est_locs, slack=1.0):
     assert true_locs.shape[-1] == est_locs.shape[-1] == 2
     assert isinstance(true_locs, torch.Tensor) and isinstance(est_locs, torch.Tensor)
 
-    locs1 = true_locs.view(-1, 2)
-    locs2 = est_locs.view(-1, 2)
+    # reshape
+    locs1 = true_locs.view(-1, 2)  # 11x2 - coordinates for true light sources
+    locs2 = est_locs.view(-1, 2)  # 7x2 - coordinates for estimated light sources
```

Comment:
Are the 11 and 7 here referring to a test case? If so, better to delete them

Reply:
Yes they're referring to a test case, I caught that a little bit ago. Will fix.